data/json/items/comestibles.json

```diff
@@ -13803,6 +13803,24 @@
         "color" : "brown",
         "addiction_type" : "nicotine",
         "spoils_in" : 0,
+        "use_action" : {
+            "type" : "consume_drug",
+            "activation_message" : "You smoke some tobacco.",
+            "effects" : [
+                {
+                    "id": "cig",
+                    "duration": 200
+                }
+            ],
+            "stat_adjustments": [
+                {"hunger" : -2},
+                {"thirst" : 1}
```

Comment:
Use of one set of stats (for example, only hunger) works fine but adding more sets (hunger + thirst, as above) breaks the process completely (so both hunger & thirst modifications are ignored and container ```stat_adjustment``` is empty).

I have no idea why this happens :(